contracts/interfaces/IChainlinkRegistry.sol

```diff
@@ -70,6 +46,32 @@ interface IChainlinkRegistry is IGovernable, ICollectableDust {
       uint256 _updatedAt,
       uint80 _answeredInRound
     );
+}
+
+interface IChainlinkRegistry is IFeedRegistry, IGovernable, ICollectableDust {
```

Comment:
I've split ChainlinkRegistry with FeedRegistry, because in mainnet we are NOT gonna save a ChainlinkRegistry, we will be saving a FeedRegistry 👼